music21/musicxml/xmlToM21.py

```diff
@@ -4498,30 +4498,41 @@ def xmlToChordSymbol(self, mxHarmony):
         '''
         # TODO: offset
         # staff is covered by insertCoreAndReference
-        r = None  # protect against undefined.
+        b: Optional[pitch.Pitch] = None
+        r: Optional[pitch.Pitch] = None
+        inversion: str = ''
+        chordKind: str = ''
+        chordKindStr: str = ''
 
         mxKind = mxHarmony.find('kind')
         if textStripValid(mxKind):
-            kindText = mxKind.text.strip()
-        else:
-            kindText = None
```

Comment:
`None` and `''` are both Falsy when evaluating kwargs in `_updateBasedOnXMLInput` so I just simplified this, but if you want to preserve `Optional[str]` we could do that. NoChord objects need an affirmative `'none'` string.